manticore/native/cpu/abstractcpu.py

```diff
@@ -89,25 +90,34 @@ class ConcretizeRegister(CpuException):
     Raised when a symbolic register needs to be concretized.
     """
 
-    def __init__(self, cpu, reg_name, message=None, policy="MINMAX"):
+    def __init__(self, cpu, reg_name, message=None, policy="MINMAX", setstate=None):
         self.message = message if message else f"Concretizing {reg_name}"
 
         self.cpu = cpu
         self.reg_name = reg_name
         self.policy = policy
+        self.setstate = setstate
```

Comment:
If you are able to set the setstate, Then can not just use a plain "Concretize" instead of a "ConcretizeRegister"

I guess this may need some documentation on why to concretize a register you need more than the cpu and the register name.

Reply:
I'll get rid of this after making changes to move the CPU back one instruction during the `ConcretizeArgument` exception handling logic. This was necessary for passing along the `setstate` function I used in `ConcretizeArgument`